statsd.py

```diff
@@ -0,0 +1,14 @@
+import tornado.web
+
+from statsd import StatsClient
+
+
+class TranscoderJob(tornado.web.RequestHandler):
+
+    @tornado.gen.coroutine
+    def post(self):
+        ###
+        # create a new job
+        ###
+        statsd.incr('transcoder_job_created')
```

Comment:
let's rename this metric to `jobs_created` since this statsd instance is already namespaced with `transcoder` :smile_cat: